# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../emacs/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Gcc-4.0 compile patch, imported from Fedora Core.

  - Rene Rebe <rene@exactcode.de>

--- emacs-21.3/lib-src/emacsclient.c~	2002-12-04 21:38:25.000000000 +0900
+++ emacs-21.3/lib-src/emacsclient.c	2005-03-18 20:21:50.542628565 +0900
@@ -38,7 +38,6 @@
 #endif /* not VMS */
 
 char *getenv (), *getwd ();
-char *getcwd ();
 
 /* This is defined with -D from the compilation command,
    which extracts it from ../lisp/version.el.  */
